pipelines/ras-rm-metrics.yml

```diff
@@ -0,0 +1,169 @@
+---
+resource_types:
+- name: cf-cli-resource
+  type: docker-image
+  source:
+    repository: nulldriver/cf-cli-resource
+    tag: latest
+
+- name: slack-notification
+  type: docker-image
+  source:
+    repository: cfcommunity/slack-notification-resource
+
+resources:
+- name: ras-rm-metrics-source
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/ras-rm-metrics.git
+    branch: deploy_app
```

Comment:
This should be pointing at master branch before merge

Reply:
@AdamHawtin I've changed it to master for merge.